app/src/main/res/layout/fragment_directory_picker.xml

```diff
@@ -1,33 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
-    android:background="@color/app_bg"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Let's use ConstraintLayout or LinearLayout here. I think we should not use RelativeLayout anywhere in the app. ConstrainLayout is more powerful and flexible. Its more performant than RelativeLayout as well.

Reply:
Sure, will make the changes and let you know.